SD-K9/src/app/components/pullup/pullup.component.scss

```
@@ -0,0 +1,17 @@
+#ionDragFooter {
+    --opacity: 0;
+}
+
+#pullup-bar {
+    z-index: 99;
+    height: 10px;
+    width: 70vw;
+    --opacity: 1;
+    --background: grey;
+    --color: grey;
+    --box-shadow: 2px 2px rgba(128, 128, 128, 0.4);
```

The pull-up bar looks good, but I am worried about usability with IOS and how close it is to the menu pull bar on iphones and newer versions of android. A user might miss and go back to their menu instead of swiping the menu up.

---

True. I can bring back the toolbar's visibility.
```suggestion
#ionDragFooter {
    --opacity: 1;
}

#pullup-bar {
    z-index: 99;
    height: 10px;
    width: 70vw;
    --opacity: 1;
    --background: grey;
    --color: grey;
    --box-shadow: 2px 2px rgba(128, 128, 128, 0.4);
```